library/commandline/src/modules/CommandLine.rb

```diff
@@ -49,33 +51,38 @@ def main
         "actions"  => {
           "help"        => {
             # translators: help for 'help' option on command line
-            "help" => _(
+            "help"     => _(
               "Print the help for this module"
-            )
+            ),
+            "readonly" => true
           },
           "longhelp"    => {
             # translators: help for 'longhelp' option on command line
-            "help" => _(
+            "help"     => _(
               "Print a long version of help for this module"
-            )
+            ),
+            "readonly" => true
           },
           "xmlhelp"     => {
             # translators: help for 'xmlhelp' option on command line
-            "help" => _(
+            "help"     => _(
               "Print a long version of help for this module in XML format"
-            )
+            ),
+            "readonly" => true
           },
           "interactive" => {
             # translators: help for 'interactive' option on command line
-            "help" => _(
+            "help"     => _(
               "Start interactive shell to control the module"
-            )
+            ),
+            "readonly" => true
```

Comment:
Please comment why `interactive` is read-only, and why `abort` is not. I would think they are the opposite.

Reply:
well, reasoning is if you start interactive mode it does not mean you do operation that need write. E.g. simple interactive and then exit it is readonly.

For abort it can be read-only, but as it is abort it always skip write. So I can add it if you think it increase readability.

Reply:
Yes, a comment would be nice. It's pretty strange, this seems as if in the interactive mode you cannot change anything...

Reply:
> well, reasoning is if you start interactive mode it does not mean you do operation that need write. E.g. simple interactive and then exit it is readonly.

Well, the question is "How do you describe what does readonly mean". I thought it means "always skip the writing step" which would break the interactive mode.

The interactive mode has `exit` (with writing) and `abort` (without writing). I am still confused.

> For abort it can be read-only, but as it is abort it always skip write. So I can add it if you think it increase readability.

I see. What about this comment then:

```rb
"abort"       => {
  # aborting is handled before writing, so "readonly" does not matter here
```